pallets/runtime/src/multisig.rs

```diff
@@ -331,6 +358,38 @@ decl_error! {
 }
 
 impl<T: Trait> Module<T> {
+    /// Private immutables
+
+    /// Add authorisation for the accountKey to become a signer of multisig
+    fn unsafe_add_auth_for_signers(
+        authorised_signer: Signatory,
+        signer: Signatory,
+        authoriser: T::AccountId,
```

Comment:
This is not needed

Reply:
We need this in the event emission

Reply:
You can extract it from `authorised_signer`. Alternatively, you can pass just this and create `authorised_signer` from this. No need to pass (especially clone) both the variables.